app/views/organisations/_expanded_works_with_organisations.html.erb

```diff
@@ -1,17 +1,13 @@
-<a href="#" class="govuk-link organisation-list__works-with-toggle" data-controls="toggle_<%= current_organisation.parameterize %>" data-expanded="false" data-toggled-text="<%= t('organisations.view_less') %>">
-  <%= t('organisations.view_all') %>
-</a>
-
-<div class="organisation-list__works-with js-hidden" id="toggle_<%= current_organisation.parameterize %>">
+<div id="toggle_<%= current_organisation.parameterize %>">
   <% works_with_organisations.each do |type, work_with_organisations| %>
     <%= render "govuk_publishing_components/components/heading", {
       text: t('organisations.type.' + type),
       heading_level: 4,
       font_size: "s",
     } %>
-    <ul class="organisation-list__works-with-list">
+    <ul class="govuk-!-margin-bottom-3">
```

Comment:
The bullets on this section are white and are announced by AT weirdly - it sort of goes _"white bullet, link"_

Updating to [`<ul class=" govuk-list govuk-list--bullet">`](https://design-system.service.gov.uk/styles/typography/) should sort.

https://user-images.githubusercontent.com/71266765/152956582-43e3cdef-9ecc-4ff7-b316-88a0ea95c644.mov